src/Combobox/index.js

```diff
@@ -113,10 +114,13 @@ const Combobox = ({
         label={label}
         startIcon={icon}
         onChange={onInputChange}
+        value={inputValue}
       />
     );
   }
 
+  const hasSelectedItem = !!selectedItem;
```

Comment:
what does the double exclamation do?

Reply:
It's a cast to boolean! Usually not needed for vanilla JS, and more handy in typed languages!

Reply:
it converts it to the truthy value, basically